Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: help build up down restart logs shell install dev test test-watch lint format check prepare typecheck publish-prep
+.PHONY: help build up down restart logs shell install dev test test-watch lint format check prepare typecheck publish-prep watch unwatch
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)